privacyidea/lib/utils.py

```diff
@@ -779,10 +752,10 @@ def parse_legacy_time(ts, return_date=False):
         return d.strftime(DATE_FORMAT)
 
 
-def parse_time_delta(s):
+def parse_timedelta(s):
```

Comment:
Do we need a safeguard for callers which expect positive timedeltas? Like an argument ``ensure_positive=True`` (as e.g. for the auth cache)?

But I guess following shit-in-shit-out would also be OK...

Reply:
Actually there was no safeguard in the past. I in the past the old method would be called with a "-7d" it would actually result in minus 7 days, since the old, deprecated method simply did an 

  int("-7")

which had the same result.

Reply:
Ah yes, you're right.